packages/response-ops/feature_flags/feature_flag_service.test.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { createFeatureFlagService } from './feature_flag_service';
+
+describe('FeatureFlagService', () => {
+  it('returns true if the feature exists', () => {
```

Comment:
I'd like to see a test using the generic - so folks know how to use it, at least minimally.  For example:

```ts
  type FeatureFlagValues = 'test.myFeature' | 'test.myOtherFeature';
  it('returns true if the feature exists (as typed)', () => {
    const featureFlagService = createFeatureFlagService<FeatureFlagValues>(['test.myFeature']);
    expect(featureFlagService.isFeatureFlagSet('test.myFeature')).toBe(true);
  });
```